src/hyperion/external_interaction/callbacks/rotation/ispyb_callback.py

```diff
@@ -61,6 +61,8 @@ def activity_gated_start(self, doc: RunStart):
                 )
                 else None
             )
+            if self.params.experiment_params.get_num_images() < 200:
+                dcgid = None 
```

Comment:
Should: I agree with codecov that this should have a test. If nothing else so that it's more obvious what the use case is